iFixit/react-commerce
# Add support for markdown in product list description

This PR closes #112 

## Changelog

- Adds `snarkdown` library to render markdown description

## QA

1. Visit the Vercel PR preview
2. Visit a few product list pages and verify that the description renders correctly
